WORKSPACE

```diff
@@ -32,11 +32,33 @@ nixpkgs_package(
     build_file = "@ai_formation_hazel//:BUILD.ghc",
 )
 
+nixpkgs_package(
+    name = "taglib",
+    repository = "@nixpkgs",
+    attribute_path = "taglib",
+    build_file_content = """
+package(default_visibility = ["//visibility:public"])
+
+filegroup (
+  name = "lib",
+  srcs = glob([
+    "lib/*.so",
+    "lib/*.so.*",
+    "lib/*.dylib",
+  ]),
+)
+""",
+)
+
 register_toolchains("@ghc//:ghc")
 
 load("//:hazel.bzl", "hazel_repositories")
 load("//:packages.bzl", "packages", "prebuilt_dependencies")
 
 hazel_repositories(
     packages=packages,
-    prebuilt_dependencies=prebuilt_dependencies)
+    prebuilt_dependencies=prebuilt_dependencies,
+    extra_libs = {
+      "tag_c": "@taglib//:lib",
```

Comment:
In some cases, we may not want to provide the library directly via Nix, but rather through a `cc_library` rule defined elsewhere in the project or in an external repository.

Could we simplify this by requiring the developer to explicitly specify either `cc_library` or `haskell_cc_import` outside of `hazel_repositories`?

For example, changing the above repository rule to directly call `haskell_cc_import`:
```
nixpkgs_package(
    name = "taglib",
    repository = "@nixpkgs",
    attribute_path = "taglib",
    build_file_content = """
package(default_visibility = ["//visibility:public"])

haskell_cc_import (
  name = "lib",
  src = glob([
    "lib/*.so",
    "lib/*.so.*",
    "lib/*.dylib",
  ])[0],
)
""",
)
```

Or
```
nixpkgs_package(
    name = "taglib",
    repository = "@nixpkgs",
    attribute_path = "taglib",
    build_file_content = """
package(default_visibility = ["//visibility:public"])

filegroup (
  name = "lib_so",
  srcs = glob([
    "lib/*.so",
    "lib/*.so.*",
    "lib/*.dylib",
  ]),
)

haskell_cc_import (
  name = "lib",
  src = ":lib_so",
)
"""
```